app/models/pds/patient.rb

```diff
@@ -0,0 +1,31 @@
+class PDS::Patient
+  include ActiveModel::Model
+
+  attr_accessor :nhs_number, :given_name, :family_name, :date_of_birth
+
+  class << self
+    def find(id)
```

Comment:
```suggestion
    def find(nhs_number)
```

Reply:
I'm of two minds on this one. On one hand, yes, this is the NHS Number, but in terms of FHIR API, this is an "id". You can see it's actually returned as "id" in the JSON response with no specific mention of "nhs_number", which is why I wasn't too bothered which one it was called here.

Still feel like it's more intuitive to call it "nhs_number" here?

Reply:
> but in terms of FHIR API, this is an "id"

I made the suggestion because upon reading the code, my first thought was – "what is the id actually". My 2p: it's fair enough that it's the ID in the FHIR API, but I think it'd be more expressive to abstract away from that and just call it what it is within our codebase.

Reply:
It's hard to gauge after working with it for a while, useful to have someone's initial reaction. 🙏

Reply:
New code reads better 👍

Reply:
New code reads better 👍